staging/src/k8s.io/client-go/scale/client_test.go

```diff
@@ -197,6 +199,20 @@ func fakeScaleClient(t *testing.T) (ScalesGetter, []schema.GroupResource) {
 				return nil, err
 			}
 			return &http.Response{StatusCode: 200, Header: defaultHeaders(), Body: bytesBody(res)}, nil
+		case "PATCH":
+			body, err := ioutil.ReadAll(req.Body)
+			if err != nil {
+				return nil, err
+			}
+			originScale, err := json.Marshal(scale)
+			if err != nil {
+				return nil, err
+			}
+			res, err := jsonpatch.MergePatch(originScale, body)
```

Comment:
this depends on the patch type being a merge patch, right? if the patch type is not a jsonmerge patch, error?

Reply:
fixed